repose-aggregator/components/services/datastore-service/datastore-service-api/src/main/java/org/openrepose/core/services/datastore/DatastoreService.java

```diff
@@ -39,6 +40,7 @@
      * @return
      * @throws DatastoreUnavailableException if no datastore exists with the given datastoreName
      */
+    // TODO: Rename this to getDistributedDatastore for clarity?
     Datastore getDatastore(String datastoreName) throws DatastoreUnavailableException;
```

Comment:
Did you mean this isn't going to strictly give you the dd?

That's how the contract read to me, which is why I left the TODO. However, we currently only ever construct one local database, and only have a way of constructing a distributed database through the API. I guess it's a matter of contract vs implementation. If we plan to support getting of a local or DD datastore in the future, then this is still a good name.

I will remove the TODO.